resources/sugarcrm/tests/test_jobwithsugarsoapprotocol.cpp

```diff
@@ -0,0 +1,298 @@
+/*
+  This file is part of FatCRM, a desktop application for SugarCRM written by KDAB.
+
+  Copyright (C) 2017 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+  Authors: Jeremy Entressangle <jeremy.entressangle@kdab.com>
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <QTest>
+#include <QDebug>
+#include <QSignalSpy>
+#include <QSettings>
+#include <Akonadi/Item>
+#include "sugaraccount.h"
+#include "sugarsession.h"
+#include "sugarsoapprotocol.h"
+#include "createentryjob.h"
+#include "updateentryjob.h"
+#include "deleteentryjob.h"
+#include "fetchentryjob.h"
+#include "accountshandler.h"
+#include "opportunitieshandler.h"
+#include "QFile"
+
+static QString username, password, host;
```

Comment:
why not member variables?

(remember to adjust the naming then, e.g. mUserName)